# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

MapperActionGoal action_goal
MapperActionResult action_result
MapperActionFeedback action_feedback
